### DISCUSSION TITLE
V# secondary color

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#7329] [Help]
i dont know how to fix this within the **tailwind.config.js** in the latest upgrade:

Filament v2 had a secondary color for many components which was gray. All references to secondary should be replaced with gray to preserve the same appearance.

Can somebody help me? Thanks in advanced.

**Comment by wychoong** [comment]
if I'm not mistaken this change is not for tailwind.config.js
search for string `secondary` in your app code and replace with `gray`
or just read the upgrade guide and use the upgrade script https://filamentphp.com/docs/3.x/panels/upgrade-guide
